tools/tags/tag-utils.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable import/no-unresolved */
+import { DA_ORIGIN } from 'https://da.live/nx/public/utils/constants.js';
+
+export const tagPathConfig = {
+  root: '/content/cq:tags',
+  ext: '.1.json',
+};
+
+export const jcrTitle = 'jcr:title';
+export const caasContentType = 'caas:content-type';
+export const caasProducts = 'caas:products';
```

Comment:
If these are specific to caas, would it be better to store them in the caas tag selector file rather than the shared utils?